extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/__init__.py

```diff
@@ -1,8 +1,9 @@
 from .local_inference.text_2_image import HuggingFaceText2ImageDiffusor
 from .local_inference.text_generation import HuggingFaceTextGenerationTransformer
-from .remote_inference_client.text_generation import HuggingFaceTextGenerationClient
+# from .remote_inference_client.text_generation import HuggingFaceTextGenerationClient
```

Comment:
Remove?

Reply:
Why does this keep erroring or everyone lol?

Reply:
will remove after rebase